### PR TITLE
Fixed #23811 -- Added a guide for using git bisect.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: docs/internals/contributing/triaging-tickets.txt [hunk 1]
```diff
@@ -420,3 +420,55 @@ the ticket database:
...
+
+.. code-block:: bash
+
+    $ ./runtests.py migrations.test_regression
+
+Next, we mark the current point in history as being "bad" since the test fails:
+
+.. code-block:: bash
+
+    $ git bisect bad
```
Comment (MarkusH) [inline docs/internals/contributing/triaging-tickets.txt]: Don't you need a `git bisect start` before (and `git bisect reset` in a dirty bisect state)?

File: docs/internals/contributing/triaging-tickets.txt [hunk 2]
```diff
@@ -420,3 +420,55 @@ the ticket database:
...
+    You need to start by "git bisect start"
+    Do you want me to do it for you [Y/n]? y
+
+Now, we need to find a point in git history before the regression was
+introduced. Use something like ``git co HEAD~100`` to checkout an earlier
+revision (100 commits earlier, in this case). Check if your test fails. If so,
+checkout an earlier revision and recheck. Once you find a revision where your
```
Comment (MarkusH) [inline docs/internals/contributing/triaging-tickets.txt]: Shouldn't this commit be marked as bad instead? Saves 100 commits from being bisected.

Comment (timgraham) [inline docs/internals/contributing/triaging-tickets.txt]: Yep

**Comment by MarkusH** [comment]
LGTM

**Pull request closed without merging**